templates/microcloud/contact-us.html

```diff
@@ -0,0 +1,153 @@
+                  <a href="tel:+4961512746816">+49 615 127 46816</a>
+                </div>
+              </div>
+            </li>
+            <li class="p-list__item u-no-padding--bottom">
+              <div class="row">
+                <div class="col-3">
+                  Japan
+                </div>
+                <div class="col-3">
+                  <a href="tel:+81362053075">+81 3 6205 3075</a>
+                </div>
+              </div>
+            </li>
+            <li class="p-list__item u-no-padding--bottom">
+              <div class="row">
+                <div class="col-3">
+                  Spain
+                </div>
+                <div class="col-3">
+                  <a href="tel:+34932201131">+34 932 201131</a>
+                </div>
+              </div>
+            </li>
+            <li class="p-list__item u-no-padding--bottom">
+              <div class="row">
+                <div class="col-3">
+                  UK and rest of the world
+                </div>
+                <div class="col-3">
+                  <a href="tel:+442036565291">+44 203 656 5291</a>
+                </div>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--25-75 is-split-on-medium">
+    <hr class="p-rule">
+    <div class="col">
+      <h2 class="p-text--small-caps">Submit a question</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-6">
+          <form action="https://ubuntu.com/marketo/submit" method="post" id="mktoForm_5364">
+            <p>Tell us about your MicroCloud use case</p>
+            <ul class="p-list">
+              <li class="p-list__item">
+                <textarea id="Comments_from_lead__c" name="Comments_from_lead__c" rows="4" maxlength="2000"></textarea>
+              </li>
+              <li class="p-list__item">
+                <label for="firstName" class="is-required">First name</label>
+                <input class="p-form-validation__input" required="" id="firstName" name="firstName" maxlength="255" type="text" />
+              </li>
+              <li class="p-list__item">
+                <label for="lastName" class="is-required">Last name</label>
+                <input class="p-form-validation__input" required="" id="lastName" name="lastName" maxlength="255" type="text" />
+              </li>
+              <li class="p-list__item">
+                <label for="email" class="is-required">Email</label>
+                <input class="p-form-validation__input" required="" id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </li>
+              <li class="p-list__item">
+                <label for="phone" class="is-required">Phone number</label>
+                <input class="p-form-validation__input" required="" id="phone" name="phone" maxlength="255" type="tel" />
```

Comment:
You may want to add [some regex](https://github.com/canonical/canonical.com/blob/main/templates/partners/partial/_homepage-form.html#L28) to at least validate it's a number. @laszlokajtar usually asks for this